webgl_generator/webgl_registry/mod.rs

```diff
@@ -88,6 +92,11 @@ impl<'a> Exts<'a> {
                     convert_html_to_doc_comment(&overview_html)
                 ),
                 new_funs,
+                min_api: match api_dep.get_attribute("version", None).unwrap() {
+                    "1.0" => Api::WebGl,
+                    "2.0" => Api::WebGl2,
```

Comment:
Hmm, I wonder if we should be smarter about parsing this... could break if a new version gets released. I guess we can fix it when it comes to that... 🤷‍♂️